backstop/tests/gdpr-banner.html

```diff
@@ -0,0 +1,79 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>GDPR banner | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/close"
+    ></script>
+
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>GDPR banner</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <div class="iui-gdpr-banner">
+        <div class="iui-gdpr-banner-title-bar">
+          <div class="iui-gdpr-banner-title">Cookie Consent</div>
+          <button
+            aria-label="Close"
+            type="button"
+            class="iui-button iui-borderless iui-small"
+          >
+            <svg-close
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-close>
+          </button>
+        </div>
+        <div class="iui-gdpr-banner-content">
+          <p>
+            We use cookies for our best experience on our website, including to personalize content and ads, to provide
+            social media features and to analyze traffic.
+            These cookies include targeted media cookies and analytics cookies. For more information see our &#32;<a
```

Comment:
```suggestion
            These cookies include targeted media cookies and analytics cookies. For more information see our&nbsp;<a
```
Our demo pages within `/minified/` remove unnecessary spaces which sometimes results in spaces that we actually want being removed.  To work around this, use `&nbsp;` to keep the space.

Reply:
Thank you, fixed 👍